app/services/cfe/compare_results.rb

```diff
@@ -17,7 +17,7 @@ def call
   private
 
     def compare_values_for(legal_aid_application)
-      v6 = CFECivil::SubmissionBuilder.new(legal_aid_application, save_result: false)
+      v6 = CFECivil::SubmissionBuilder.new(legal_aid_application)
       v6.call
```

Comment:
**nit** and assuming it works fine to use class method direct

```suggestion
      CFECivil::SubmissionBuilder.call(legal_aid_application)
```

Reply:
1) This is also a stub that will no longer be called after cutover and is due for deletion in the very near future and;
2) No, Because in the next line we pass the v6 object, containing the response to the comparison tool. Making the call object return the CFE Result invalidated it's use in the main flow!